src/validators/BirthdateInputValidator.js

```diff
@@ -56,20 +56,18 @@ export const getMinDateValidator = (props) => {
 }
 
 export const validateMinMaxDateFactory = (props) => {
-  const { minAge, maxAge, dateFormat, customErrorMessage } = props
+  const {
+    minBirthdate,
+    maxBirthdate,
+    minAge,
```

Comment:
So we only need these for the messaging huh? Bummer. But I can't think of another way.

Reply:
I was thinking of calculating the age from the supplied `minBirthdate` and maxBirthdate` , but I thought it might just be cleaner to pass in the ages you want to be displayed.